README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Getting Started
 
-This project is a [_fastlane_](https://github.com/fastlane/fastlane) plugin. To get started with `fastlane-plugin-browserstack`, add it to your project by running:
+This project is a [_fastlane_](https://github.com/fastlane/fastlane) plugin. To get started with `browserstack-fastlane-plugin`, add it to your project by running:
```

Comment:
This is fastlane-plugin-browserstack as you can refer https://rubygems.org/gems/fastlane-plugin-browserstack